src/web/extractors.rs

```diff
@@ -1091,6 +1092,26 @@ impl HawkIdentifier {
     }
 }
 
+impl TryFrom<&HttpRequest> for HawkIdentifier {
+    type Error = Error;
+
+    fn try_from(req: &HttpRequest) -> Result<HawkIdentifier, Error> {
+        if DOCKER_FLOW_ENDPOINTS.contains(&req.uri().path().to_lowercase().as_str()) {
+            return Ok(HawkIdentifier::cmd_dummy());
+        }
+        let method = req.method().clone();
+        // NOTE: `connection_info()` gets a mutable reference lock on `extensions()`, so
```

Comment:
Looking at what goes on here, the sequence is all blocking code. Is the concern that we don't want the risk of something modifying the `extensions`?